### PR TITLE
add new feature: add more state types to get striked through ticket n…

### PR DESCRIPTION
**Pull request by reneeb**
…umber in link table

### CODE REVIEW

File: Kernel/Output/HTML/LinkObject/Ticket.pm [hunk 1]
```diff
@@ -304,7 +309,8 @@ sub TableCreateSimple {
         return;
     }
 
-    my $TicketHook = $Kernel::OM->Get('Kernel::Config')->Get('Ticket::Hook');
+    my $ConfigObject = $Kernel::OM->Get('Kernel::Config');
```
Comment (mgruner) [inline Kernel/Output/HTML/LinkObject/Ticket.pm]: Superfluous?

File: Kernel/Output/HTML/LinkObject/Ticket.pm [hunk 2]
```diff
@@ -324,7 +330,9 @@ sub TableCreateSimple {
 
                 # set css
                 my $CssClass;
-                if ( $Ticket->{StateType} eq 'merged' ) {
+                my @StatesToStrike = @{ $Self->{ConfigObject}->Get('LinkTicket::StrikeThroughStates') || [] };
```
Comment (mgruner) [inline Kernel/Output/HTML/LinkObject/Ticket.pm]: Broken? $Self->{ConfigObject}

File: Kernel/Config/Files/Ticket.xml [hunk 3]
```diff
@@ -11470,4 +11470,14 @@ Thanks for your help!
             </Option>
         </Setting>
     </ConfigItem>
+    <ConfigItem Name="LinkTicket::StrikeThroughStates" Required="0" Valid="1">
```
Comment (mgruner) [inline Kernel/Config/Files/Ticket.xml]: This setting should be added on a logically good place with other similar settings.

**Comment by mgruner** [comment]
Hi @reneeb, can you please check and update this?

**Comment by reneeb** [comment]
sure...

**Comment by reneeb** [comment]
replaced by #981

**Pull request closed without merging**